src/tinkerio.py

```diff
@@ -360,7 +360,7 @@ def calltinker(self, command, stdin=None, print_to_screen=False, print_command=F
                 if len(vw.split('.')) <= 2:
                     vn = float(vw)
                 else:
-                    vn = float(vw.split('.')[:2])
+                    vn = float(vw.split('.')[0]) + 0.1*float(vw.split('.')[1])
```

Comment:
I think this is a fix for the known bug that @leeping mentioned in the slack chat. @leeping, to ensure that there's a paper trail, could you edit the PR description to include something like
```
- [x] Fixes <description of issue here>
```

Reply:
This issue was reported by a student in a private message but I don't think there's an issue corresponding to it.